src/main/resources/META-INF/plugin.xml

```diff
@@ -1,71 +1,86 @@
-       Guidelines: https://plugins.jetbrains.com/docs/marketplace/plugin-overview-page.html#plugin-name -->
-  <name>MapStructPlugin</name>
+    <!-- Public plugin name should be written in Title Case.
+         Guidelines: https://plugins.jetbrains.com/docs/marketplace/plugin-overview-page.html#plugin-name -->
+    <name>DoStruct</name>
 
-  <!-- A displayed Vendor name or Organization ID displayed on the Plugins Page. -->
-  <vendor email="info@techconative.com" url="https://www.techconative.com">TechConative</vendor>
+    <!-- A displayed Vendor name or Organization ID displayed on the Plugins Page. -->
+    <vendor email="info@techconative.com" url="https://www.techconative.com">TechConative</vendor>
+
+    <version>0.0.4</version>
+
+    <idea-version since-build="222" until-build="231.*"/>
 
     <change-notes>><![CDATA[
-   <b>How to Use The Plugin</b><br>
+   <b>Simple and easy steps to use the plugin</b><br>
 
       - Select the xml code.<br>
       - press Ctrl + Shift  + ` shortcut  and provide the details.<br>
       - Code will be generated at the specified location.<br>
-      - Can run in any platform independent of OS.
-      - Can just view generated code without writing.
-      - Can get code in pop up for already created class and append the missing part.
-      - you can go to action by pressing (⌘+Shift+A or Ctrl+Shift+A) and type plugin name and change the shortcut yourself.
+      - Can run in any platform independent of OS.<br>
+      - Can just view generated code without writing.<br>
+      - Can get code in pop up for already created class and append the missing part.<br>
+      - you can go to action by pressing (⌘+Shift+A or Ctrl+Shift+A) and type plugin name
+      and change the shortcut yourself.<br>
+
+    <b>Newly added</b><br>
+
+      - Just right click on selected text and select <i>GenerateMapping</i>  which appears first on popup menu.<br>
+
 
   ]]>
     </change-notes>
 
-  <!-- Description of the plugin displayed on the Plugin Page and IDE Plugin Manager.
-       Simple HTML elements (text formatting, paragraphs, and lists) can be added inside of <![CDATA[ ]]> tag.
-       Guidelines: https://plugins.jetbrains.com/docs/marketplace/plugin-overview-page.html#plugin-description -->
-  <description><![CDATA[
-    This is a productivity tool to be used when you're modernizing you're moving the mappings from Dozer to Mapstruct in your applications.
+    <!-- Description of the plugin displayed on the Plugin Page and IDE Plugin Manager.
+         Simple HTML elements (text formatting, paragraphs, and lists) can be added inside of <![CDATA[ ]]> tag.
+         Guidelines: https://plugins.jetbrains.com/docs/marketplace/plugin-overview-page.html#plugin-description -->
+    <description><![CDATA[
+    DoStruct is a productivity tool that one can leverage at the time of migrating to Mapstruct in spring based application that's using legacy Dozer for mappings.
```

Comment:
Few corrections,
- DoStruct is not a `code-genaration` tool.
- The application need not be spring based at all. Neither Dozer, nor Mapstruct needs Spring to operate.

Reply:
Fixed.